signedjson/types.py

```diff
@@ -56,7 +59,5 @@ def sign(self, message):
 
     @property
     def verify_key(self):
-        # Note: use `signedjson.key.get_verify_key` to get a
-        # `signedjson.types.VerifyKey`.
```

Comment:
Was this not valid?

Reply:
It was! I reverted too eagerly. Serves me right for squashing.